code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -1,3 +1,5 @@
+var/list/intercomm_list = list()

```

Comment:
This list is unused. remove.

Reply:
Right, originally intended to play stationwide.